third_party/cachetools/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
```

Comment:
Looks like the code has been directly added. You have to instead add the gitmodule. This is similar to how you have added the google auth library. See that in `third_party` directory, it should look like a reference `cachetools @ <commit>`

Reply:
I might need some help with this